content/rippled-api-methods/server_info.md

```diff
@@ -280,6 +280,7 @@ The `info` object may have some arrangement of the following fields:
 
 | `Field`                             | Type                      | Description |
 |:------------------------------------|:--------------------------|:-----------|
+| `amendment_blocked`                 | Boolean                   | If the server is [amendment blocked](concept-amendments.html#amendment-blocked), this field displays and is set to `true`. If the server is older than 0.80.0, this field does not display even if the server is amendment blocked. If the server is not amendment blocked, this field does not display. |
```

Comment:
Add a _(May be omitted)_ marker, and rephrase to match the "closed_ledger" phrasing a little more closely, and use a badge for the new feature. Namely:

> If `true`, this server is [amendment blocked] (concept-amendments.html#amendment-blocked). If the server is not amendment blocked, the response omits this field. [New in: rippled 0.80.0][] |

(You'll also have to add 0.80.0 to `snippets/rippled_versions.md`)